cpp/benchmarks/flow_simulation_ode_seir.cpp

```diff
@@ -32,16 +32,16 @@ namespace mio
 namespace benchmark
 {
 
-using FlowModel = ::mio::oseir::Model;
+using FlowModel = ::mio::oseir::Model<ScalarType>;
 
 using namespace oseir;
 
 // For comparison benchmarks, an old model version that does not provide computation of flows has been reimplemented here.
 // For more details see the original implementation in:
 // https://github.com/SciCompMod/memilio/blob/13555a6b23177d2d4633c393903461a27ce5762b/cpp/models/ode_seir/model.h
-class FlowlessModel : public CompartmentalModel<InfectionState, Populations<InfectionState>, Parameters>
+class FlowlessModel : public CompartmentalModel<ScalarType,InfectionState, Populations<ScalarType, InfectionState>, Parameters<ScalarType>>
```

Comment:
This benchmark model currently does not compile, I suppose it missed out on the whole typename/template treatment. Should the comment mention these edits?